### ISSUE TITLE
Docker Alpine not starting - zip error

### ISSUE THREAD

**Issue by rabesocke** [scm-manager/scm-manager#2053]
## Issue description

Since the new version with Alpine and Debian Docker image, it is not possible for me to update to the latest tag. The Alpine version doesn't start because of a zip error.

#### Describe how to reproduce the bug

---

#### Describe your SCM-Manager setup

---

#### Which operating system, browser and versions do you use?

Docker version `:latest` since version 2.34

#### Log of the bug

```
2022-06-02 19:54:29.949:INFO::main: Logging initialized @565ms to org.eclipse.jetty.util.log.StdErrLog
2022-06-02 19:54:30.584:INFO:oejs.Server:main: jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 11.0.15+10-alpine-r0
2022-06-02 19:54:30.868:WARN:oejw.WebAppContext:main: Failed startup of context o.e.j.w.WebAppContext@68267da0{/scm,null,STOPPED}{./var/webapp/scm-webapp.war}
java.util.zip.ZipException: invalid entry CRC (expected 0x966dbfc3 but got 0x66e606be)
	at java.base/java.util.zip.ZipInputStream.readEnd(Unknown Source)
	at java.base/java.util.zip.ZipInputStream.read(Unknown Source)
	at java.base/java.util.jar.JarInputStream.read(Unknown Source)
	at org.eclipse.jetty.util.IO.copy(IO.java:174)
	at org.eclipse.jetty.util.IO.copy(IO.java:124)
	at org.eclipse.jetty.util.resource.JarResource.copyTo(JarResource.java:218)
	at org.eclipse.jetty.webapp.WebInfConfiguration.unpack(WebInfConfiguration.java:644)
	at org.eclipse.jetty.webapp.WebInfConfiguration.preConfigure(WebInfConfiguration.java:143)
	at org.eclipse.jetty.webapp.WebAppContext.preConfigure(WebAppContext.java:488)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:523)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.server.Server.start(Server.java:423)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
	at org.eclipse.jetty.server.Server.doStart(Server.java:387)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at sonia.scm.server.ScmServer.init(ScmServer.java:130)
	at sonia.scm.server.ScmServer.run(ScmServer.java:91)
	at sonia.scm.server.ScmServerDaemon.main(ScmServerDaemon.java:53)
2022-06-02 19:54:30.989:INFO:oejw.StandardDescriptorProcessor:main: NO JSP Support for /, did not find org.eclipse.jetty.jsp.JettyJspServlet
2022-06-02 19:54:31.020:INFO:oejs.session:main: DefaultSessionIdManager workerName=node0
2022-06-02 19:54:31.021:INFO:oejs.session:main: No SessionScavenger set, using defaults
2022-06-02 19:54:31.027:INFO:oejs.session:main: node0 Scavenging every 660000ms
2022-06-02 19:54:31.122:INFO:oejsh.ContextHandler:main: Started o.e.j.w.WebAppContext@7b36aa0c{/,[file:///opt/scm-server/var/webapp/docroot/],AVAILABLE}
2022-06-02 19:54:31.178:INFO:oejs.AbstractConnector:main: Started ServerConnector@add0edd{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
2022-06-02 19:54:31.186:INFO:oejs.Server:main: Started @1815ms
```

**Comment by pfeuffer** [comment]
Hey @rabesocke ,

this is strange. Indeed we changed the Docker builds with 2.34.0. Nonetheless, I can still run the alpine images on my Ubuntu PC, on our k8s servers and on my small Synology DiskStation. May you tell us, what your host OS is?

Thanks
René

**Comment by rabesocke** [comment]
Hi @pfeuffer,

the last running version for me is 2.33.0. All Docker updates after this version have the same problems (Alpine as well as Ubuntu).

My host system is a OMV server (Ubuntu 10).

Best regards
Christian

**Comment by rabesocke** [comment]
The version 2.36.0 still has the same problem. The Debian image is working fine after adding `wget`. But I would use the Alpine image because that is connected to the latest tag.

**Comment by pfeuffer** [comment]
Your issue was reported by another user who is running SCM-Manager on a virtulized server. Are you using some kind of virtualization, too, or is docker running "on bare metal" on Ubuntu?

**Comment by rabesocke** [comment]
Hi @pfeuffer, I have no virtualized server running. I have a HP MicroServer with an OpenMediaVault Debian image running directly on the "bare metal".

**Comment by pfeuffer** [comment]
Hi @rabesocke , we just pushed a docker image based on an updated alpine image with an SCM-Manager 2.36.0 (the fix makes it a 2.36.1 technically). It would be great if you could test, whether this one starts on your machine. It's not necessary to link it to your data, it's just interesting, whether the CRC error is gone.

The docker image is availabe with this tag: `cloudogu/scm-manager:2.36.1-9955db5ee-3`

Thank you very much 🙂

**Comment by rabesocke** [comment]
Hi @pfeuffer, looks good. I just started a container with this image without linking any volumes or ports.

```
2022-06-21 10:35:50.789:INFO::main: Logging initialized @639ms to org.eclipse.jetty.util.log.StdErrLog
2022-06-21 10:35:51.824:INFO:oejs.Server:main: jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 11.0.15+10-alpine-r1
2022-06-21 10:35:55.076:INFO:oejw.StandardDescriptorProcessor:main: NO JSP Support for /scm, did not find org.eclipse.jetty.jsp.JettyJspServlet
2022-06-21 10:35:55.118:INFO:oejs.session:main: DefaultSessionIdManager workerName=node0
2022-06-21 10:35:55.119:INFO:oejs.session:main: No SessionScavenger set, using defaults
2022-06-21 10:35:55.124:INFO:oejs.session:main: node0 Scavenging every 660000ms
2022-06-21 10:35:55.721 [main] [          ] INFO  sonia.scm.lifecycle.classloading.ClassLoaderLifeCycle - create new simple ClassLoaderLifeCycle
2022-06-21 10:35:55.735 [main] [          ] INFO  sonia.scm.lifecycle.BootstrapContextFilter - register for restart events
2022-06-21 10:35:55.741 [main] [          ] INFO  sonia.scm.event.LegmanScmEventBus - create new event bus ScmEventBus-1
2022-06-21 10:35:55.896 [main] [          ] WARN  io.micrometer.core.instrument.binder.jvm.ExecutorServiceMetrics - Failed to bind as org.apache.shiro.concurrent.SubjectAwareExecutorService is unsupported.
2022-06-21 10:35:55.899 [main] [          ] INFO  sonia.scm.event.ScmEventBus - use sonia.scm.event.LegmanScmEventBus as event bus implementation
2022-06-21 10:35:56.018 [main] [          ] INFO  sonia.scm.lifecycle.BootstrapContextListener - start scm-manager initialization
2022-06-21 10:35:56.896 [main] [          ] WARN  sonia.scm.lifecycle.PluginBootstrap - install plugin scm-git-plugin:2.36.1-SNAPSHOT
2022-06-21 10:35:57.161 [main] [          ] WARN  sonia.scm.lifecycle.PluginBootstrap - install plugin scm-hg-plugin:2.36.1-SNAPSHOT
2022-06-21 10:35:57.347 [main] [          ] WARN  sonia.scm.lifecycle.PluginBootstrap - install plugin scm-svn-plugin:2.36.1-SNAPSHOT
2022-06-21 10:35:57.571 [main] [          ] WARN  sonia.scm.lifecycle.PluginBootstrap - install plugin scm-legacy-plugin:2.36.1-SNAPSHOT
2022-06-21 10:35:57.613 [main] [          ] INFO  sonia.scm.plugin.PluginProcessor - collect plugins
2022-06-21 10:35:57.680 [main] [          ] INFO  sonia.scm.plugin.PluginProcessor - install plugin tree:
+- scm-git-plugin@2.36.1-SNAPSHOT
+- scm-legacy-plugin@2.36.1-SNAPSHOT
+- scm-hg-plugin@2.36.1-SNAPSHOT
+- scm-svn-plugin@2.36.1-SNAPSHOT

2022-06-21 10:35:59.785 [main] [          ] INFO  sonia.scm.update.repository.XmlRepositoryV1UpdateStep - no v1 repositories database file found
2022-06-21 10:36:00.501 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.repository.xml and version 2.0.0 (class sonia.scm.update.repository.XmlRepositoryV1UpdateStep)
2022-06-21 10:36:00.502 [main] [          ] INFO  sonia.scm.update.repository.XmlRepositoryV1UpdateStep - no v1 repositories database file found
2022-06-21 10:36:00.563 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.dao.xml and version 2.0.0 (class sonia.scm.update.store.DifferentiateBetweenConfigAndConfigEntryForGlobalStoreUpdateStep)
2022-06-21 10:36:00.573 [main] [          ] INFO  sonia.scm.update.store.DifferentiateBetweenConfigAndConfigEntryUpdateStep - Updating config entry file: /var/lib/scm/config/executedUpdates.xml
2022-06-21 10:36:00.867 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin-center and version 2.0.0 (class sonia.scm.update.plugin.PluginCenterUpdateStep)
2022-06-21 10:36:00.901 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin.svn and version 2.0.0 (class sonia.scm.repository.SvnV2UpdateStep)
2022-06-21 10:36:00.952 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.user.xml and version 2.0.0 (class sonia.scm.update.user.XmlUserV1UpdateStep)
2022-06-21 10:36:00.956 [main] [          ] INFO  sonia.scm.update.user.XmlUserV1UpdateStep - no v1 file for users found
2022-06-21 10:36:00.978 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin.PluginSetsConfig and version 2.0.0 (class sonia.scm.update.plugin.PluginSetsConfigInitializationUpdateStep)
2022-06-21 10:36:01.008 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.git.lfs and version 2.0.0 (class sonia.scm.web.lfs.LfsV1UpdateStep)
2022-06-21 10:36:01.026 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.index and version 2.0.0 (class sonia.scm.update.index.RemoveCombinedIndex)
2022-06-21 10:36:01.054 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.group.xml and version 2.0.0 (class sonia.scm.update.group.XmlGroupV1UpdateStep)
2022-06-21 10:36:01.055 [main] [          ] INFO  sonia.scm.update.group.XmlGroupV1UpdateStep - no v1 file for groups found
2022-06-21 10:36:01.071 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin.git and version 2.0.0 (class sonia.scm.repository.update.GitV2UpdateStep)
2022-06-21 10:36:01.097 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin-center and version 2.0.1 (class sonia.scm.update.plugin.PluginCenterJreVersionUpdateStep)
2022-06-21 10:36:01.138 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.security.xml and version 2.0.1 (class sonia.scm.update.security.XmlSecurityV1UpdateStep)
2022-06-21 10:36:01.152 [main] [          ] INFO  sonia.scm.update.security.XmlSecurityV1UpdateStep - no v1 file for security found
2022-06-21 10:36:01.175 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.git.lfs and version 2.0.1 (class sonia.scm.web.lfs.RemoveRepositoryIdFromBlobStoreUpdateStep)
2022-06-21 10:36:01.193 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.repository.xml and version 2.0.1 (class sonia.scm.update.repository.XmlRepositoryFileNameUpdateStep)
2022-06-21 10:36:01.223 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.repository.xml and version 2.0.2 (class sonia.scm.update.repository.MigrateVerbsToPermissionRoles)
2022-06-21 10:36:01.370 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.repository.xml and version 2.0.3 (class sonia.scm.update.repository.PublicFlagUpdateStep)
2022-06-21 10:36:01.371 [main] [          ] INFO  sonia.scm.update.repository.PublicFlagUpdateStep - Migrating public flags of repositories as RepositoryRolePermission 'READ' for user '_anonymous'
2022-06-21 10:36:01.404 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type config.xml and version 2.4.0 (class sonia.scm.update.repository.AnonymousModeUpdateStep)
2022-06-21 10:36:01.439 [main] [          ] INFO  sonia.scm.update.UpdateEngine - running update step for type sonia.scm.plugin-center.authentication and version 2.30.0 (class sonia.scm.update.plugin.PluginCenterAuthenticationUpdateStep)
2022-06-21 10:36:01.545 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.web.SvnServletModule
2022-06-21 10:36:01.546 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.autoconfig.AutoConfigModule
2022-06-21 10:36:01.546 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.web.HgServletModule
2022-06-21 10:36:01.546 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.security.gpg.GPGModule
2022-06-21 10:36:01.546 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.web.GitServletModule
2022-06-21 10:36:01.547 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.repository.hooks.HookModule
2022-06-21 10:36:01.547 [main] [          ] INFO  sonia.scm.lifecycle.modules.ApplicationModuleProvider - add module class sonia.scm.legacy.LegacyModule
2022-06-21 10:36:01.610 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.metrics.HttpMetricsFilter to filter chain
2022-06-21 10:36:01.611 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.filter.BaseUrlFilter to filter chain
2022-06-21 10:36:01.612 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.security.TokenExpiredFilter to filter chain
2022-06-21 10:36:01.613 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.legacy.LegacyProtocolServletAuthenticationFilter to filter chain
2022-06-21 10:36:01.617 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.web.filter.DefaultHttpProtocolServletAuthenticationFilter to filter chain
2022-06-21 10:36:01.620 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.web.security.ApiAuthenticationFilter to filter chain
2022-06-21 10:36:01.622 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.filter.MDCFilter to filter chain
2022-06-21 10:36:01.623 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind filter class sonia.scm.web.security.TokenRefreshFilter to filter chain
2022-06-21 10:36:01.626 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind servlet class sonia.scm.lifecycle.RestartServlet to servlet chain
2022-06-21 10:36:01.627 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind servlet class sonia.scm.web.i18n.I18nServlet to servlet chain
2022-06-21 10:36:01.631 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind servlet class sonia.scm.web.protocol.HttpProtocolServlet to servlet chain
2022-06-21 10:36:01.633 [main] [          ] INFO  sonia.scm.filter.WebElementModule - bind servlet class sonia.scm.WebResourceServlet to servlet chain
2022-06-21 10:36:01.737 [main] [          ] INFO  sonia.scm.plugin.DefaultExtensionProcessor - start processing extensions
2022-06-21 10:36:01.804 [main] [          ] WARN  sonia.scm.plugin.ExtensionBinder - could not find extension for extension point interface sonia.scm.io.FileSystem
2022-06-21 10:36:01.858 [main] [          ] WARN  sonia.scm.plugin.ExtensionBinder - could not find extension for extension point interface sonia.scm.api.v2.resources.LogoutRedirection
2022-06-21 10:36:02.064 [main] [          ] INFO  sonia.scm.plugin.DefaultExtensionProcessor - bound extensions in 325.9 ms
2022-06-21 10:36:02.473 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.initialization.InitializationRealm
2022-06-21 10:36:02.476 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.web.security.AdministrationContextRealm
2022-06-21 10:36:02.477 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.legacy.LegacyRealm
2022-06-21 10:36:02.477 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.security.ApiKeyRealm
2022-06-21 10:36:02.478 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.security.AnonymousRealm
2022-06-21 10:36:02.478 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.security.DefaultRealm
2022-06-21 10:36:02.479 [main] [          ] INFO  sonia.scm.lifecycle.modules.ScmSecurityModule - bind security realm class sonia.scm.security.BearerRealm
Jun 21, 2022 10:36:12 AM com.google.inject.servlet.GuiceFilter setPipeline
WARNING: Multiple Servlet injectors detected. This is a warning indicating that you have more than one GuiceFilter running in your web application. If this is deliberate, you may safely ignore this message. If this is NOT deliberate however, your application may not work as expected.
2022-06-21 10:36:12.644 [main] [          ] WARN  sonia.scm.cache.GuavaCacheConfigurationReader - could not find manual configuration at /var/lib/scm/ext/gcache.xml
2022-06-21 10:36:12.715 [main] [          ] WARN  sonia.scm.repository.HgVerifier - /opt/java/openjdk/bin/hg is not a regular file
2022-06-21 10:36:12.715 [main] [          ] WARN  sonia.scm.repository.HgVerifier - /usr/local/sbin/hg is not a regular file
2022-06-21 10:36:12.716 [main] [          ] WARN  sonia.scm.repository.HgVerifier - /usr/local/bin/hg is not a regular file
2022-06-21 10:36:12.716 [main] [          ] WARN  sonia.scm.repository.HgVerifier - /usr/sbin/hg is not a regular file
2022-06-21 10:36:14.066 [main] [          ] INFO  sonia.scm.repository.DefaultRepositoryManager - added RepositoryHandler class sonia.scm.repository.HgRepositoryHandler for type Type{name=hg, displayName=Mercurial}
2022-06-21 10:36:14.542 [main] [          ] INFO  sonia.scm.repository.DefaultRepositoryManager - added RepositoryHandler class sonia.scm.repository.SvnRepositoryHandler for type Type{name=svn, displayName=Subversion}
2022-06-21 10:36:14.543 [main] [          ] INFO  sonia.scm.repository.DefaultRepositoryManager - added RepositoryHandler class sonia.scm.repository.GitRepositoryHandler for type Type{name=git, displayName=Git}
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.DefaultBranchDeleteProtection
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.plugin.PluginCenterRefresh
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.user.AnonymousUserDeletionEventHandler
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.user.DefaultUserManager
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.EventDrivenRepositoryArchiveCheckInitializer
2022-06-21 10:36:15.060 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.LastModifiedUpdateListener
2022-06-21 10:36:15.061 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.api.v2.resources.GitRepositoryConfigChangeClearRepositoryCacheListener
2022-06-21 10:36:15.061 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.RemoveDeletedRepositoryRole
2022-06-21 10:36:15.061 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.security.AuthorizationChangedEventProducer
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.ReadOnlyCheckInitializer
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.spi.FileLockPreCommitHook
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.update.RepositoryUpdateAfterCreationHook
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.DefaultBranchChangedDispatcher
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.config.ScmConfigurationChangedListener
2022-06-21 10:36:15.063 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.DefaultRepositoryRoleManager
2022-06-21 10:36:15.064 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.repository.GitRepositoryModifyListener
2022-06-21 10:36:15.065 [main] [          ] INFO  sonia.scm.lifecycle.modules.EagerSingletonModule - initialize eager singleton sonia.scm.initialization.DefaultInitializationFinisher
2022-06-21 10:36:15.114 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ====================================================
2022-06-21 10:36:15.114 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ==                                                ==
2022-06-21 10:36:15.114 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ==    Startup token for initial user creation     ==
2022-06-21 10:36:15.114 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ==                                                ==
2022-06-21 10:36:15.114 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ==              5p6drYexqLu0BqJJOvkV              ==
2022-06-21 10:36:15.115 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ==                                                ==
2022-06-21 10:36:15.115 [main] [          ] WARN  sonia.scm.lifecycle.AdminAccountStartupAction - ====================================================
2022-06-21 10:36:15.386 [main] [          ] INFO  sonia.scm.repository.work.WorkdirProvider - deleting 0 old work dirs in /tmp/scm-work
2022-06-21 10:36:15.762 [main] [          ] INFO  sonia.scm.web.security.AdministrationContextRealm - assign admin permissions to admin context user _scmsystem
2022-06-21 10:36:15.767 [main] [          ] INFO  sonia.scm.group.DefaultGroupManager - create group _authenticated of type xml
2022-06-21 10:36:16.010 [ScmEventBus-1-2] [          ] INFO  sonia.scm.security.DefaultAuthorizationCollector - invalidate cache, because of a received authorization event
2022-06-21 10:36:16.073:INFO:oejsh.ContextHandler:main: Started o.e.j.w.WebAppContext@5df2023c{SCM-Manager ${project.version},/scm,file:///var/cache/scm/work/webapp/webapp/,AVAILABLE}{./var/webapp/scm-webapp.war}
2022-06-21 10:36:16.092:INFO:oejw.StandardDescriptorProcessor:main: NO JSP Support for /, did not find org.eclipse.jetty.jsp.JettyJspServlet
2022-06-21 10:36:16.094:INFO:oejsh.ContextHandler:main: Started o.e.j.w.WebAppContext@6029f2a4{/,[file:///opt/scm-server/var/webapp/docroot/],AVAILABLE}
2022-06-21 10:36:16.182:INFO:oejs.AbstractConnector:main: Started ServerConnector@add0edd{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
2022-06-21 10:36:16.182:INFO:oejs.Server:main: Started @26054ms
2022-06-21 10:36:20.840 [qtp724125922-15] [          ] INFO  sonia.scm.lifecycle.modules.ResteasyAllInOneServletDispatcher - init resteasy
```

**Comment by pfeuffer** [comment]
Thats great, thank you very much for your help!

**Comment by pfeuffer** [comment]
We released version `2.36.1` with this fix.
